templates/composer.yml

```diff
@@ -118,6 +118,12 @@ objects:
                 optional: true
           - name: SPLUNK_HEC_PORT
             value: "${SPLUNK_HEC_PORT}"
+          - name: GLITCHTIP_DSN
+            valueFrom:
+              secretKeyRef:
+                key: dsn
```

Comment:
`glitchtip_dsn` for consistency with the env variable name? `dsn` isn't super-descriptive.

Reply:
this is the secret imported from the namespace, so we do not have control over the key name, but I can add comments to explain DSN.

Reply:
Ahhh, makes sense. :) Thanks for the explanation.